notebooks/enterprise-search/app-search-engine-exporter.ipynb

```diff
@@ -470,29 +594,34 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# by default we are adding a `sparse_vector` field for all text fields in our engine\n",
-    "# feel free to modify this list to only include the fields that are relevant\n",
-    "SPARSE_VECTOR_FIELDS = [\n",
-    "    field_name + \"_semantic\" for field_name in schema if schema[field_name] == \"text\"\n",
-    "]\n",
-    "\n",
-    "sparse_vector_fields = {}\n",
-    "for field_name in SPARSE_VECTOR_FIELDS:\n",
-    "    # this is added so we can use semantic search with ELSER\n",
-    "    sparse_vector_fields[field_name] = {\"type\": \"sparse_vector\"}\n",
-    "\n",
-    "elasticsearch.indices.put_mapping(index=DEST_INDEX, properties=sparse_vector_fields)"
+    "# delete our inference endpoint if it is already created\n",
+    "if elasticsearch.inference.get(inference_id=\"elser_inference_endpoint\"):\n",
+    "    elasticsearch.inference.delete(inference_id=\"elser_inference_endpoint\")\n",
+    "\n",
+    "# and create our endpoint using the ELSER v2 model\n",
+    "elasticsearch.inference.put(\n",
+    "    inference_id='elser_inference_endpoint',\n",
+    "    inference_config={\n",
+    "        \"service\": \"elasticsearch\",\n",
+    "        \"service_settings\": {\n",
+    "            \"model_id\": \".elser_model_2_linux-x86_64\",\n",
```

Comment:
Should we add a note that this will not work on aarch64? Or is that moot because we're using cloud for the examples?

Reply:
Yeah - since it's cloud - it's a moot point.